app/src/main/java/rs/urosdragojevic/emotionsters_android/MainActivity.java

```diff
@@ -33,25 +42,46 @@ protected void onCreate(Bundle savedInstanceState) {
         Call<Homescreen> call = jsonPlaceHolderApi.getHomescreen();
 
         call.enqueue(new Callback<Homescreen>() {
+            @SuppressLint("SetTextI18n")
             @Override
-            public void onResponse(Call<Homescreen> call, Response<Homescreen> response) {
+            public void onResponse(@NonNull Call<Homescreen> call, @NonNull Response<Homescreen> response) {
                 if (!response.isSuccessful()) {
                     textView.setText("Code: " + response.code());
                     textView1.setText("Code: " + response.code());
                     return;
                 }
                 Homescreen homescreen = response.body();
-
-                    textView.setText(homescreen.getText());
-                    textView1.setText(homescreen.getText1());
-
+                textView.setText(homescreen.getText());
+                textView1.setText(homescreen.getText1());
```

Comment:
Text and Text1 are not good variable names, think of better names

Reply:
Done